main/com.spotify/eval/train/annotated.txt

```diff
@@ -862,3 +862,48 @@ UT: @org.thingpedia.media-source.playlist() filter id =~ "billy" => @org.thingpe
 U: put vienna song on my playlist bill
 UT: $dialogue @org.thingpedia.dialogue.transaction.execute;
 UT: @org.thingpedia.media-source.song() filter id =~ "vienna" => @com.spotify.add_song_to_playlist(playlist="bill", song=id);
+====
+# 1
```

Comment:
By convention, manually written developer data has IDs that start with `manual/`